### DISCUSSION TITLE
2008/07/30/%E5%BC%80%E5%A7%8B%E5%8A%A8%E7%AC%94%E5%86%99%E4%B9%A6%E4%BA%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#233] [Announcements]
# 2008/07/30/%E5%BC%80%E5%A7%8B%E5%8A%A8%E7%AC%94%E5%86%99%E4%B9%A6%E4%BA%86

上星期正式跟出版社签了约。交稿时间是明年三月，字数50万字。开始听到这个字数吓了我一跳，我以为这就是Word中显示出来的字数要五十万。累死我也写不出来这么多字啊。后来才打听明白，书籍出版的字数不是这么算的。它的算法很简单，基板上一页纸（16开）按1400～1600字计算，乘以页数，就是总字数。所以50万字的意思也就是最后出版的书在350页左右。我翻开身边几本计算机类的书籍一算，果然是这么回事。要是按照Word的统计方法，字数也许就只有一半了。而我写LabVIEW，程序片段都是图片，算不了字数，Word统计起来就更低了。

https://blog.qizhen.xyz/2008/07/30/%E5%BC%80%E5%A7%8B%E5%8A%A8%E7%AC%94%E5%86%99%E4%B9%A6%E4%BA%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

亚东
2008年12月19日 13:28 编辑
期待……最好能让我做教材，哈哈

回复
Qizhen
2008年09月12日 10:43 编辑
跟我联系的编辑很热情的。我也没有什么特别的要求。所以我俩就确定了一下书的内容、字数、交稿日期等就签约了。都是email联系的，所以也没谈太多。

回复
Hyper
2008年09月11日 14:18 编辑
 请问跟出版社谈要注意什么, 不方便的话请回email. 多谢

回复
Mary
2008年08月11日 22:22 编辑
期待高手出书中，到时是否能得到本签名珍藏版阿？

回复
Tom
2008年08月6日 20:18 编辑
《RuanQizhen品LabVIEW》

回复
Qizhen
2008年08月4日 09:36 编辑
照这起名方式还能叫《LabVIEW那些事》，《相约LabVIEW》，《LabVIEW姐夫》，《LabVIEW都是活雷锋》:)

回复
杰
2008年08月3日 21:20 编辑
《不得不说的事——我和我的LabVIEW》
《那年，那月，那山，那人，那美丽的LabVIEW》
《狂人日记之——LabVIEW》
《Lab和VIEW的秘密爱情故事》
《十八禁！LabVIEW》（估计这名字能卖的最火）
哈哈哈，，，，，，，，，，，，，，，，，，，，
 

回复
Tom
2008年08月2日 20:18 编辑
可以叫：
LabVIEW Cookbook
Effective LabVIEW
Professional LabVIEW
LabVIEW in Action
LabVIEW Program Design
Thinking in LabVIEW (LabVIEW编程思想)
LabVIEW Strategies and Tactics
The Art of LabVIEW
Practical LabVIEW
LabVIEW Hacks
LabVIEW Programming by Example
LabVIEW编程指南
LabVIEW高级编程技巧与实例
…

回复
yoyo
2008年08月1日 18:04 编辑
好牛掰阿 加油写阿

回复
秋
2008年08月1日 16:35 编辑
可以~

回复
ruirui
2008年08月1日 14:23 编辑
这个书名,一定得配合一个大照片印在封面才行呀:)

回复
zomby
2008年07月31日 20:33 编辑
恭喜！一点要买上几本，除了自己保留，还得介绍给一些朋友。
但愿印刷数可以多点，清华大学那本《软件工程方法在LabVIEW中的应用》在广州购书中心已经没有存货了。

回复
言佳
2008年07月31日 20:10 编辑
终于等到你要出书了，到时我一定要买两本！
加油！LabVIEW方面的书还不怎么多，希望你的书中能更多地介绍一些高级应用。

回复
颖
2008年07月31日 17:34 编辑
终于要出书了哦，恭喜！

回复
xiaoji
2008年07月31日 12:12 编辑
革命尚未成功，同志仍需努力 Fighting！！

回复
li
2008年07月30日 22:24 编辑
看BLOG就等于是先拜读了大作…..

回复
Zhen
2008年07月30日 22:03 编辑
期待啊
到时候再搞个签名售书火一把，哈哈~

回复
HA
2008年07月30日 19:57 编辑
厉害厉害~

回复
派
2008年07月30日 19:29 编辑
恭喜恭喜，到时候又有大作可以欣赏了。

回复
Kisha
2008年07月30日 19:28 编辑
签约给钱吗

回复
